src/Spain/Mews.Fiscalizations.Spain.Tests/SoapFaultResponseDeserializationTests.cs

```diff
@@ -44,9 +45,9 @@ public async Task InvalidSoapActionShouldFail()
         {
             var soapFault = await soapClient.SendAsync<InvalidSoapMessage, SubmitIssuedInvoicesResponse>(new InvalidSoapMessage { Message = "bla-bla-bla" });
             Assert.IsFalse(soapFault.IsSuccess, "Soap fail was expected but success received");
-            Assert.AreEqual(soapFault?.ErrorResult?.Code, "env:Client");
-            var message = soapFault?.ErrorResult?.Message;
-            Assert.IsTrue(soapFault?.ErrorResult?.Message?.Contains("El XML no cumple el esquema.") ?? false, $"Expected fragment not found in message {message}");
+            Assert.AreEqual(soapFault.ErrorResult.Get().Code, "env:Client");
+            var message = soapFault.ErrorResult.Map(e => e.Message);
```

Comment:
You use Get on line 48 and then map on 49. Either just do `var error = soapFault.ErrorResult.Get()` or use map on all the places.